collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/ResetProjectTest.kt

```diff
@@ -3,67 +3,76 @@ package org.odk.collect.android.feature.settings
-        rule.startAtMainMenu()
+    fun canResetBlankForms() {
+        rule.startAtFirstLaunch()
+            .clickTryCollect()
+            .copyForm("all-widgets.xml")
             .openProjectSettingsDialog()
             .clickSettings()
             .clickProjectManagement()
             .clickOnResetApplication()
-            .assertText(R.string.reset_settings_dialog_title)
-            .assertDisabled(R.string.reset_settings_button_reset)
-            .rotateToLandscape(ResetApplicationDialog())
-            .assertText(R.string.reset_settings_dialog_title)
-            .assertDisabled(R.string.reset_settings_button_reset)
-            .rotateToPortrait(ResetApplicationDialog())
-            .assertText(R.string.reset_settings_dialog_title)
             .assertDisabled(R.string.reset_settings_button_reset)
+            .clickOnString(R.string.reset_blank_forms)
+            .clickOnString(R.string.reset_settings_button_reset)
+            .clickOKOnDialog(MainMenuPage())
+            .clickFillBlankForm()
+            .assertTextDoesNotExist("All widgets")
     }
 
     @Test
-    fun savedAndBlankForms_shouldBeReset() {
-        rule.startAtMainMenu()
-            .copyForm("all-widgets.xml")
-            .startBlankForm("All widgets")
-            .clickGoToArrow()
-            .clickJumpEndButton()
-            .clickSaveAsDraft()
-            .clickDrafts()
-            .assertText("All widgets")
-            .pressBack(MainMenuPage())
+    fun canResetSavedFormsAndEntities() {
+        testDependencies.server.addForm("one-question-entity-registration.xml")
+        testDependencies.server.addForm(
+            "one-question-entity-update.xml",
+            listOf(EntityListItem("people.csv"))
+        )
+
+        rule.withMatchExactlyProject(testDependencies.server.url)
+            .startBlankForm("One Question Entity Registration")
+            .fillOutAndFinalize(FormEntryPage.QuestionAndAnswer("Name", "Logan Roy"))
+
             .openProjectSettingsDialog()
             .clickSettings()
             .clickProjectManagement()
             .clickOnResetApplication()
             .assertDisabled(R.string.reset_settings_button_reset)
             .clickOnString(R.string.reset_saved_forms)
-            .clickOnString(R.string.reset_blank_forms)
             .clickOnString(R.string.reset_settings_button_reset)
             .clickOKOnDialog(MainMenuPage())
-            .clickFillBlankForm()
-            .assertTextDoesNotExist("All widgets")
+
+            .clickDrafts()
+            .assertTextDoesNotExist("One Question Entity Registration")
             .pressBack(MainMenuPage())
-            .clickDrafts(false)
-            .assertTextDoesNotExist("All widgets")
+
+            .startBlankForm("One Question Entity Update")
+            .assertTextDoesNotExist("Logan Roy")
     }
 
     @Test
-    fun adminSettings_shouldBeReset() {
-        rule.startAtMainMenu()
+    fun canResetAdminSettings() {
```

Comment:
I think the terminology we have been using is `unprotected`/`protected` instead of `user` (or whatever)/`admin` settings.